linters/trufflehog/plugin.yaml

```
@@ -0,0 +1,34 @@
+version: 0.1
+lint:
+  downloads:
+    - name: trufflehog
+      downloads:
+        - os:
+            linux: linux
+            macos: darwin
+          cpu:
+            x86_64: amd64
+            arm_64: arm64
+          url: https://github.com/trufflesecurity/trufflehog/releases/download/v${version}/trufflehog_${version}_${os}_${cpu}.tar.gz
+  definitions:
+    - name: trufflehog
+      files: [ALL]
+      download: trufflehog
+      known_good_version: 3.31.3
+      commands:
+        - name: lint
+          output: sarif
+          run: trufflehog filesystem --json --fail --no-verification ${target}
+          read_output_from: stdout
+          success_codes: [0, 183]
+          is_security: true
+          parser:
+            runtime: python
+            run: ${plugin}/linters/trufflehog/trufflehog_to_sarif.py
```

```suggestion
            run: python ${plugin}/linters/trufflehog/trufflehog_to_sarif.py
```
Shebangs won't work on windows, so it will need the python prefix.

---

landing as-is because this change caused an error - tyler is looking into it now

---

This works with python3 under certain other circumstances. Documented the bug and working on a fix